docs/releasing.md

```diff
@@ -0,0 +1,27 @@
+# Releasing
+
+
+To create a release manually there are two steps:
+
+
+1. version bumping
+2. create & publish distribution
+
```

Comment:
maybe add  "update the changelog" or will we automate that / make it optional?

Reply:
The idea is that you update the changelog in every PR to the main branch, and not wait untill the version bump. Could indeed try to automate this, but I think that is not worth it atm

Reply:
made PR template to remind you when opening a PR